src/javascripts/views/homeView.js

```diff
@@ -1,7 +1,85 @@
+import Template from "../templates/template";
```

Comment:
Add empty line after import block

Reply:
I fixed it @phuocduong-agilityio  . Please help me check. Thanks !